shacl/generate_shacl.py

```diff
@@ -0,0 +1,60 @@
+from rdflib import Graph, Literal, BNode, URIRef
+from rdflib.collection import Collection
+
+import sys
+sys.path.append('..')
+from bricksrc.namespaces import RDF, RDFS, BRICK, BSH, SH, SKOS
+from bricksrc.namespaces import bind_prefixes
+# add SHACL shapes from properties
+from bricksrc.properties import properties as property_definitions
+
+G = Graph()
+bind_prefixes(G)
+A = RDF.type
+
+for name, properties in property_definitions.items():
+
+    for pred, obj in properties.items():
+        sh_prop = BNode()
+
+        # We are only concerned with properties that have RDFS.domain or .range
+        # predicate.  The temporary replacements for those predicates are
+        # brick:expectedDomain and :expectedRange.  See properties.py for
+        # explanation.
```

Comment:
Remember to change `RDFS.domain` to `BRICK.expectedDomain` and so on for `range`

Reply:
Yeah,  two more properties have crept in this week.  Playing the catch up work...